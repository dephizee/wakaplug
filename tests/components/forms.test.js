import React from 'react';
import renderer from 'react-test-renderer';
import { InputField, FormContainer} from "./../../components/forms.style";


test('InputField working', () => {
  const component = renderer.create(
    <InputField/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('FormContainer working', () => {
    const component = renderer.create(
      <FormContainer><InputField/></FormContainer>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });