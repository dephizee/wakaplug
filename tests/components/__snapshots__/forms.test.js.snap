// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormContainer working 1`] = `
<form
  className="sc-gsDKAQ kpUtDf"
>
  <input
    className="sc-bdvvtL eEqMGc"
    minLength="1"
  />
</form>
`;

exports[`InputField working 1`] = `
<input
  className="sc-bdvvtL eEqMGc"
  minLength="1"
/>
`;
